back-end/common/models/persons.py

```diff
@@ -17,3 +18,52 @@ class Meta:
 
     def __str__(self):
         return self.full_name
+
+    
+class Relative(Person):
+    relative_list = [('father', 'отец'),
+                    ('mother', 'мать'),
+                    ('sister', 'сестра'),
+                    ('brother', 'брат')]
+    rel_status = models.CharField(max_length=32, choices=relative_list)
```

Comment:
It may be better to use `TextChoices` here. [Reference](https://docs.djangoproject.com/en/3.1/ref/models/fields/#choices).